pyserver/tilemaptown_server/server.py

```diff
@@ -136,6 +136,8 @@ async def clientHandler(websocket, path):
 		print("disconnected: %s (%s, \"%s\")" % (client.ip, client.username or "?", client.name))
 	except:
 		print("Unexpected error:", sys.exc_info()[0])
+		print(sys.exc_info()[1])
```

Comment:
This is a much better solution than my `raise` which you can see here commented out